rust/agama-lib/share/profile.schema.json

```diff
@@ -302,11 +302,15 @@
                 "title": "The name of the network interface bound to this connection",
                 "type": "string"
               },
-              "macAddress": {
+              "CustomMacAddress": {
```

Comment:
maybe we should write the first letter lower? `customMacAddress`

Reply:
yep, it should be customMacAddress, human error ;)